AmplifyCombineSupport/AmplifyCombineSupportTests/StorageCombineSupportChainTests.swift

```diff
@@ -0,0 +1,102 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import XCTest
+
+import AmplifyCombineSupport
+
+@testable import Amplify
+@testable import AmplifyTestCommon
+
+class ChainTests: XCTestCase {
```

Comment:
`class StorageCombineSupportChainTests` and also don't see this file added to target

Reply:
Same problem w/file renaming. Removed